packages/react/src/toggle/useToggle.ts

```diff
@@ -38,18 +38,16 @@ export function useToggle(parameters: useToggle.Parameters): useToggle.ReturnVal
 
   const getRootProps = React.useCallback(
     (externalProps?: GenericHTMLProps): GenericHTMLProps => {
-      return mergeReactProps(
-        externalProps,
-        {
+      return getButtonProps(
```

Comment:
#1445 is now merged, so you can write `mergeReactProps(getButtonProps, externalProps, {...})`